user/src/main/java/com/chronica/user/logic/security/SecurityConfiguration.java

```diff
@@ -0,0 +1,52 @@
+package com.chronica.user.logic.security;
+
+import com.chronica.user.data.constant.Api;
+import lombok.AllArgsConstructor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+import org.springframework.security.web.servlet.util.matcher.MvcRequestMatcher;
+import org.springframework.web.servlet.handler.HandlerMappingIntrospector;
+
+@Configuration
+@AllArgsConstructor
+@EnableWebSecurity
+public class SecurityConfiguration {
+
+    private final FilterBeforeRequest filterBeforeRequest;
+
+    @Bean
+    public SecurityFilterChain securityFilterChain(HttpSecurity http, HandlerMappingIntrospector introspector) throws Exception {
+        MvcRequestMatcher.Builder mvcMatcherBuilder = new MvcRequestMatcher.Builder(introspector);
+
+        http.addFilterBefore(filterBeforeRequest, UsernamePasswordAuthenticationFilter.class).csrf(csrfConfigurer ->
+                csrfConfigurer.ignoringRequestMatchers(mvcMatcherBuilder.pattern(Api.ACCOUNT))
```

Comment:
Jak masz jedną metodę w lambdzie to czytelniej byłoby to zrobić tak:
 http.addFilterBefore(filterBeforeRequest, UsernamePasswordAuthenticationFilter.class)
       .csrf(csrfConfigurer ->csrfConfigurer.ignoringRequestMatchers(mvcMatcherBuilder.pattern(Api.ACCOUNT)));